app/payment-link-v2/reference-confirm/reference-confirm.njk

```diff
@@ -1,13 +1,49 @@
 {% extends "../../views/layout-payment-links-v2.njk" %}
-{% from "govuk/components/button/macro.njk" import govukButton %}
-{% from "govuk/components/input/macro.njk" import govukInput %}
 {% from "govuk/components/back-link/macro.njk" import govukBackLink %}
-{% from "../../views/macros/error-summary.njk" import errorSummary %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/inset-text/macro.njk" import govukInsetText %}
 
 {% block pageTitle %}
-  {{ __p('paymentLinksV2.referenceConfirm.areYouSureThisIsAValid') }} {{product.reference_label}}  - {{ productName }}
+  {{ isThisAValidReferenceText }}  - {{ productName }}
 {% endblock %}
 
 {% block contentBody %}
-  <p class="govuk-body">Reference Confirm page - placeholder text</p>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-l">{{ productName }}</h1>  
+
+    {% if productDescription %}
+      <p class="govuk-body">{{ productDescription }}</p>
+    {% endif %}
+
+    <h2 class="govuk-heading-m">{{ paymentReferenceLabel }}</h2>
+
+    {% if paymentReferenceHint %}
+      <p class="govuk-hint">{{ paymentReferenceHint }}</p>    
+    {% endif %}
+
+    {{ govukInsetText({
+      text: "4242424242424242",
```

Comment:
Good spot - forgot to mention that in the PR.\
Seems like in the Design system, they usually have the Primary button at the start - so I thought it may be better that ways.  

Please let me know what you think?

<img width="949" alt="image" src="https://user-images.githubusercontent.com/59831992/156615490-5e05387a-733d-4f01-b0ce-015d1d2d725b.png">